description.txt

```diff
@@ -1 +1,29 @@
-This is a description file for git training session, now resolve conflicts 
\ No newline at end of file
+This is a description file for git training sessiThis is a description file for git training session
```

Comment:
@lakshyagkmit branch naming conventions are not followed here, going forward kindly follow the same
it could be `feature-session-mom`, `docs-session-mom`